app/src/main/java/com/github/gotify/service/WebSocketConnection.java

```diff
@@ -57,6 +69,7 @@ synchronized WebSocketConnection onClose(Runnable onClose) {
     }
 
     synchronized WebSocketConnection onOpen(Runnable onOpen) {
+        reconnectHandler.removeCallbacks(reconnectCallback);
```

Comment:
onOpen() will only be executed when the connection will be created in WebSocketService#startPushService. so this line should have no effect.